### PR TITLE
Format readme into category sections

### PR DESCRIPTION
**Pull request by JuanAr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,48 @@
...
+## Cards
+These examples emphasize the rich card support in Bot Framework.
+
+Sample | Description | C# | Node
+------------ | ------------- | :-----------: | :-----------:
+Rich Cards | A sample bot to renders several types of cards as attachments. | [View Sample](/CSharp/cards-RichCards)[![Deploy to Azure][Deploy Button]][Deploy CSharp/RichCards] | [View Sample](/Node/cards-RichCards)[![Deploy to Azure][Deploy Button]][Deploy Node/RichCards]
+Carousel of Cards | A sample bot that sends multiple rich card attachments in a single message using the Carousel layout. | [View Sample](/CSharp/cards-CarouselCards)[![Deploy to Azure][Deploy Button]][Deploy CSharp/CarouselCards] | [View Sample](/Node/cards-CarouselCards)[![Deploy to Azure][Deploy Button]][Deploy Node/CarouselCards]
+
+## Intelligence
+Build bots with powerful algorithms leveragin intelligence APIs.
```
Comment (ejadib) [inline README.md]: Typo in leveragin**g**

File: README.md [hunk 2]
```diff
@@ -1,25 +1,48 @@
...
+
+Sample | Description | C# | Node
+------------ | ------------- | :-----------: | :-----------:
+LUIS | A sample bot using LuisDialog to integrate with a LUIS.ai application. | [View Sample](/CSharp/intelligence-LUIS)[![Deploy to Azure][Deploy Button]][Deploy CSharp/LUIS] | [View Sample](/Node/intelligence-LUIS)[![Deploy to Azure][Deploy Button]][Deploy Node/LUIS]
+
+## Demo
+These are bots designed to showcase what's possible on specific channels. They're great sources of code fragments if you're looking to have you bot lightup specific features for a channel.
```
Comment (ejadib) [inline README.md]: Mmmm... we are not showing channel specifics features in those channels. I would rephrase this to say something like these are more complex, end-to-end, samples or something like that.

File: README.md (hunk 1 above)
Comment (pcostantini) [inline README.md]: "leveraging"

File: README.md (hunk 2 above)
Comment (pcostantini) [inline README.md]: if you're looking to have you_r_ bot lightup specific

**Comment by JuanAr** [comment]
Feedback applied!
